brayns/Brayns.cpp

```diff
@@ -324,81 +324,55 @@ struct Brayns::Impl : public PluginAPI
-                        geometryParameters.getCircuitReport()});
-        pm.setProperty({"circuitMeshFolder", "circuitMeshFolder",
-                        geometryParameters.getCircuitMeshFolder()});
-        pm.setProperty({"circuitRandomSeed", "circuitRandomSeed",
-                        int32_t(geometryParameters.getCircuitRandomSeed())});
-        pm.setProperty({"circuitDensity", "circuitDensity",
-                        geometryParameters.getCircuitDensity()});
-        pm.setProperty({"circuitUseSimulationModel",
-                        "circuitUseSimulationModel",
-                        geometryParameters.getCircuitUseSimulationModel()});
-        pm.setProperty({"circuitMeshFilenamePattern",
-                        "circuitMeshFilenamePattern",
-                        geometryParameters.getCircuitMeshFilenamePattern()});
-        pm.setProperty({"radiusMultiplier", "radiusMultiplier",
-                        double(geometryParameters.getRadiusMultiplier())});
-        pm.setProperty({"radiusCorrection", "radiusCorrection",
-                        double(geometryParameters.getRadiusCorrection())});
-        pm.setProperty({"colorScheme", "colorScheme",
-                        enumToString(geometryParameters.getColorScheme()),
-                        enumNames<ColorScheme>()});
-        pm.setProperty({"geometryQuality", "geometryQuality",
-                        enumToString(geometryParameters.getGeometryQuality()),
-                        enumNames<GeometryQuality>()});
-        {
-            const auto layout = geometryParameters.getMorphologyLayout();
-
-            pm.setProperty({"morphologyLayoutNbColumns",
-                            "morphologyLayoutNbColumns",
-                            int32_t(layout.nbColumns),
-                            {0, std::numeric_limits<int32_t>::max()}});
-
-            pm.setProperty({"morphologyLayoutVerticalSpacing",
-                            "morphologyLayoutVerticalSpacing",
-                            int32_t(layout.verticalSpacing),
-                            {0, std::numeric_limits<int32_t>::max()}});
-
-            pm.setProperty({"morphologyLayoutHorizontalSpacing",
-                            "morphologyLayoutHorizontalSpacing",
-                            int32_t(layout.horizontalSpacing),
-                            {0, std::numeric_limits<int32_t>::max()}});
+            pm.addProperty({"boundingBoxP0", "boundingBoxP0", p0});
+            pm.addProperty({"boundingBoxP1", "boundingBoxP1", p1});
         }
-
+        pm.addProperty({"dampenBranchThicknessChangerate",
+                         "dampenBranchThicknessChangerate",
+                         geometryParameters
+                             .getMorphologyDampenBranchThicknessChangerate()});
+        pm.addProperty({"useSDFGeometries", "useSDFGeometries",
+                         geometryParameters.getMorphologyUseSDFGeometries()});
+        pm.addProperty({"loadCacheFile", "loadCacheFile",
```

Comment:
cache still here?

Reply:
Caching appears at multiple places. I prefer to have a dedicated PR for dealing with that.